src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
```

Comment:
We don't refer to the console as an SDK, its just the `Console` or `Appwrite Console` depending on the context.